Neos.Neos/Resources/Private/Translations/de/Modules.xlf

```diff
@@ -204,7 +204,7 @@
         <target state="final">In diesem Arbeitsbereich gibt es keine unveröffentlichten Änderungen.</target>
       </trans-unit>
       <trans-unit id="workspaces.discardSelectedChangesInWorkspaceConfirmation" xml:space="preserve" approved="yes">
-        <source>Do you really want to discard selected changes in the "{0}" workspace?</source>
+        <source>Do you really want to discard the selected changes in the "{0}" workspace?</source>
```

Comment:
I'm not a native speaker and might be wrong but IMO the wording was better before. Using "the" in this case is not necessary and sounds choppy to me

Reply:
😄 no native speaker myself but I suggested usigng "the" because it sounds right to me.

Reply:
IIRC it's a typical German fallacy ;)

Reply:
🤔 it would be consistent with all other texts in this document 🤷‍♀️

Reply:
did you mean "all _the_ other texts"? ;)

Reply:
It's probably both fine